metacat-common-server/src/main/java/com/netflix/metacat/common/server/partition/parser/ASTNULL.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *    Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
2020?

Reply:
Kept it consistent with what we have in other files.